api/app/config.py

```diff
@@ -1,40 +1,34 @@
 from functools import lru_cache
 
-import httpx
-from pydantic import BaseSettings, Field, HttpUrl, validator
+from pydantic import BaseSettings, Field
 
 TMDB_API_URL = "https://api.themoviedb.org/3"
 
 
 class Settings(BaseSettings):
 
     tmdb_api_url: str = TMDB_API_URL
-    tmdb_api_key: str = Field(..., env="TMDB_API_TOKEN")
-    tmdb_base_path: HttpUrl | None = None
-
-    @validator("tmdb_base_path", always=True)
-    def set_tmdb_base_path(cls, v, values):
-        """Assign the tmdb_base_path"""
-
-        # we use a validator here to "compute" tmdb_base_path, which requires tmdb_api_key
-        # see: https://pydantic-docs.helpmanual.io/usage/validators/#validate-always
-
-        # get configuration for poster base_url (e.g.: https://image.tmdb.org/t/p/)
-        #  https://api.themoviedb.org/3/configuration?api_key=<<api_key>>
-
-        tmdb_config_url = f"{values['tmdb_api_url']}/configuration"
-        config_resp = httpx.get(
-            tmdb_config_url, params={"api_key": values["tmdb_api_key"]}
-        )
-        return config_resp.json()["images"]["secure_base_url"]
+    tmdb_api_key: str = Field(
+        ...,
+        env="TMDB_API_TOKEN",
+        description="From https://www.themoviedb.org/settings/api",
+    )
+    secret: str = Field(
+        ..., env="SECRET_KEY", description="A unique unpredictable value"
+    )
+
+    # security
+    cookie_secure: bool = Field(False, env="COOKIE_SECURE")
```

Comment:
should switch these to default to being secure, and patching them/overriding them in tests & localdev